camelot/model/party.py

```diff
@@ -416,7 +433,13 @@ class Admin(GeographicBoundary.Admin):
 
         field_attributes = {h:copy.copy(v) for h,v in GeographicBoundary.Admin.field_attributes.items()}
         attributes_dict = {
-            'code': {'name': _('Postal code')},
+            'code': {
+                'name': _('Postal code'),
+                'tooltip': lambda c: zip_code_types[c.zip_code_type].tooltip if c.zip_code_type is not None else None,
+                'validator_type': ZipcodeValidator.__name__,
+                'validator_state': ZipcodeValidator.state_for_city,
+                'background_color': lambda c: ColorScheme.VALIDATION_ERROR if not ZipcodeValidator.for_city(c).validity(c.code).valid else None,
```

Comment:
Afhankelijk van hoe we de validators op termijn gaan omzetten in C++  vermoed ik dat idealiter de tooltip (validatie hint) en de achtergrondkleur automatisch afgeleid/opgevraagd moeten kunnen worden op basis van het validatieresultaat?

Reply:
dat zou eigenlijk al zo moeten zijn ... normaal gezien moet je geen background color meegeven en gaat die oranje kleuren als invalid

tooltip op basis van regexp is vermoedelijk bizar :)

Reply:
Hmm, dat moet ik nog eens te goei testen dan. Want zo zat het ook in mijn hoofd, maar ik dacht dat dat nu enkel voor bepaalde editors gebeurde, niet voor de plain text.

Wat betreft de tooltip, daar doelde ik eerder op het feit dat de validator op termijn validatie-hints kunnen geven (al dan niet in de tooltip).

Reply:
mja, haal die bg-color er wel uit aub, want die gaat nu bij elke aanroep een QValidator construeren ...

Reply:
Fixed in https://github.com/v-finance/camelot/pull/609.